### PR TITLE
doc: added kyc onboarding

### PR DESCRIPTION
**Pull request by mighty840**
## Motivation and Context

The KYC onboarding page on the documentation was empty. The KYC onboarding is a critical element for businesses and users to be compliant with anti-money laundering laws. The ETOPay SDK offers a strong option of onboarding users using two KYC providers. Using the ETOPay application dashboard, it is possible to configure the parameters and let ETOPay do the heavy-lifting. Since the KYC onboarding flows are already integrated in the SDK, it makes sense to inform customers on how to use this feature. 

## Summary

- Updated the KYCOnboard.md
- Added relevant flows and links to external documents.

## Checklist

Please ensure that your PR meets the following requirements:

### General

- [ ] ~Code follows project style and guidelines.~
- [ ] ~No redundant or duplicate code.~
- [ ] ~No added new dependencies without clear justification and approval.~

### Documentation

- [X] Added/updated relevant documentation (e.g., README, inline comments).
- [ ] ~Added a CHANGELOG entry for major changes.~

### Testing

- [ ] ~Added relevant test cases, leveraging tools like `rstest` or similar for reusable tests.~
- [ ] Tests pass locally.

### Build & CI/CD

- [ ] Confirmed no CI/CD pipeline issues.
- [ ] ~Updated build scripts where necessary.~

## Comments to Reviewer

Please provide any comments or points of consideration for the reviewers, such as specific areas of focus, potential edge cases, or context that would aid the review process.

### Reviewer Responsibilities

- [ ] Code adheres to style and guidelines.
- [ ] All tests are appropriately covered and they pass.
- [ ] Consider performance and/or security impacts.
- [ ] Constructive feedback is provided.

### CODE REVIEW

**Review by hauju — APPROVED** [review]

File: docs/docs/SDK Modules/KYC Onboarding.md [hunk 1]
```diff
@@ -1,2 +1,120 @@
...
+
+3. **Mitigating Fraud and Scams** – The Web3 space has seen numerous scams, including rug pulls, Ponzi schemes, and fraudulent token offerings. By implementing KYC, projects can verify the legitimacy of users and project teams, fostering trust within the ecosystem.
+
+4. **Protecting Institutional Investors** – Institutional investors are increasingly exploring Web3 and DeFi. However, without proper KYC measures, they face significant risks. KYC-compliant platforms provide greater security and legitimacy, encouraging broader institutional participation.
+
+## Onboarding users for KYC
+
+Currently, the ETOPay SDK offers two alternatives for onboarding users through a KYC process. These two processes are indepedent of each other, however the ETOPay SDK consults the backend to check which process the user has started and also continues from the state where the user previously left. hence, in case if the user loses their devices, or re-initiates the onboarding through another client of the ETOPay SDK, the user is guided onward from where they left of, instead of starting from the beginning.
```
Comment (antbern) [inline docs/docs/SDK Modules/KYC Onboarding.md:23]:
```suggestion
Currently, the ETOPay SDK offers two alternatives for onboarding users through a KYC process. These two processes are indepedent of each other, however the ETOPay SDK consults the backend to check which process the user has started and also continues from the state where the user previously left. Hence, in case if the user loses their devices, or re-initiates the onboarding through another client of the ETOPay SDK, the user is guided onward from where they left of, instead of starting from the beginning.
```

File: docs/docs/SDK Modules/KYC Onboarding.md [hunk 2]
```diff
@@ -1,2 +1,120 @@
...
+
+The two KYC processes that are offered by the SDK are namely:
+
+1. Using Postident as the KYC Provider
```
Comment (antbern) [inline docs/docs/SDK Modules/KYC Onboarding.md:27]: Should we link to Postident here?

File: docs/docs/SDK Modules/KYC Onboarding.md [hunk 3]
```diff
@@ -1,2 +1,120 @@
...
+
+3. **Mitigating Fraud and Scams** – The Web3 space has seen numerous scams, including rug pulls, Ponzi schemes, and fraudulent token offerings. By implementing KYC, projects can verify the legitimacy of users and project teams, fostering trust within the ecosystem.
+
+4. **Protecting Institutional Investors** – Institutional investors are increasingly exploring Web3 and DeFi. However, without proper KYC measures, they face significant risks. KYC-compliant platforms provide greater security and legitimacy, encouraging broader institutional participation.
+
+## Onboarding users for KYC
+
+Currently, the ETOPay SDK offers two alternatives for onboarding users through a KYC process. These two processes are indepedent of each other, however the ETOPay SDK consults the backend to check which process the user has started and also continues from the state where the user previously left. hence, in case if the user loses their devices, or re-initiates the onboarding through another client of the ETOPay SDK, the user is guided onward from where they left of, instead of starting from the beginning.
+
+The two KYC processes that are offered by the SDK are namely:
```
Comment (antbern) [inline docs/docs/SDK Modules/KYC Onboarding.md:25]:
```suggestion
The two KYC processes that are offered by the SDK are:
```

File: docs/docs/SDK Modules/KYC Onboarding.md [hunk 4]
```diff
@@ -1,2 +1,120 @@
...
+
+    Currently, the federation of the KYC status of the user across multiple customers is not supported. This would mean that the same user might need to perform a KYC again for a different ETOPay customer. This feature is highly demanded as it improves the user onboarding, and it provides new ETOPay customers immediate access to a verified user pool. However, the feature is currently under scrutiny and is bound to be planned in future releases of the ETOPay platform.
+
+!!! warning
+
+    There is a provider lock, that currently prevents users from switching between the two providers, or in future, potentially multiple providers, once they chose to start the process with a certain provider. Due to this, the user is forced to complete the onboarding with the same provider they chose at the beginning and have no option of switching providers once a process is started.
+
+### Onboarding users via Postident
+
+For onboarding users on KYC via Postident allows users with existing wallets to simply perform a KYC onboarding, migrate the mnemonic to ETOPay and start using the wallet services directly.
```
Comment (antbern) [inline docs/docs/SDK Modules/KYC Onboarding.md:47]: This sentence has a weird structure: "For onboarding [...] via Postident allows [...]". Can you clarify?

File: docs/docs/SDK Modules/KYC Onboarding.md [hunk 5]
```diff
@@ -1,2 +1,120 @@
...
+
+Before being able to use the Postident integration in the SDK for all the end-users, the customers should register with Postident and get a business account for using their services. The SDK is merely a wrapper around the Postident Standard Connect and Response (SCR) interface for faster onboarding without the need of programming own integrations. After creating a business account, the Postident provides information as shown in the table below.
+
+There are typically two access credentials provided. One access to their `TestApp` on the ITU can be plugged in to the ETOPay test environment. The other access to their production can be plugged in to the ETOPay production environment.
```
Comment (antbern) [inline docs/docs/SDK Modules/KYC Onboarding.md:51]: Should we use the wording `live` and `test` instead of test and prod?

File: docs/docs/SDK Modules/KYC Onboarding.md [hunk 6]
```diff
@@ -1,2 +1,120 @@
...
+
+???+ info
+
+    The Postident also publishes their own SDK for both Android and iOS platforms. These SDKS work in conjunction with the ETOPay SDK. The SDK's from Postident provide the UI/UX for the KYC Onboarding, whilst the ETOPay SDK handles the business logic of generating case Ids, tracking them and securely encrypting. Simply feeding the case Ids generated by the ETOPay SDK to the Postident SDK provides a complete all rounded KYC Integration for end-users with a simplified UI/UX
```
Comment (antbern) [inline docs/docs/SDK Modules/KYC Onboarding.md:55]:
```suggestion
    Postident also publishes their own SDK for both Android and iOS platforms. These SDK's work in conjunction with the ETOPay SDK. The SDK's from Postident provide the UI/UX for the KYC Onboarding, whilst the ETOPay SDK handles the business logic of generating case Ids, tracking them and securely encrypting. Simply feeding the case Ids generated by the ETOPay SDK to the Postident SDK provides a complete all rounded KYC Integration for end-users with a simplified UI/UX
```

File: docs/docs/SDK Modules/KYC Onboarding.md [hunk 7]
```diff
@@ -1,2 +1,120 @@
...
+|50|	YES|	general, personal, identity, residence, amla & documents|	699 EUR|	20.000 EUR|	there’s a criteria with increased risk for this user|
+|40|	YES|	general, personal, identity, residence, amla & documents|	14.999 EUR|	250.000 EUR |	no specific risk criteria|
+|30|	YES|	general, personal, identity, residence, amla & documents|	250.000 EUR|	250.000 EUR|	full verification plus source of funds document|
+|20|	YES|	general, personal, identity, residence, amla & documents|	individual|	individual| |
+|10|	YES|	general, personal, identity, residence, amla & documents|	individual|	individual|	|
+
+Users can further lower their risk level by providing more information about themselves. For example, to set the risk level to 60 the exchange needs information about the country of origin and the person (general and personal kyc step). To achieve a risk level of 40, all KYC steps (general, personal, identity, residence, amla & documents) must be completed. To achieve an even lower risk level, the exchange requires further individual information from the user. To do this, the user can directly contact [support](support@etospheres.com).
```
Comment (antbern) [inline docs/docs/SDK Modules/KYC Onboarding.md:120]:
```suggestion
Users can further lower their risk level by providing more information about themselves. For example, to set the risk level to 60 the exchange needs information about the country of origin and the person (general and personal kyc step). To achieve a risk level of 40, all KYC steps (general, personal, identity, residence, amla & documents) must be completed. To achieve an even lower risk level, the exchange requires further individual information from the user. To do this, the user can directly contact [support](mailto:support@etospheres.com).
```

**Review by antbern — COMMENTED** [review]
Very nice! Added some small typo / style fixes 💯

File: docs/docs/SDK Modules/KYC Onboarding.md (hunk 2 above)
Comment (mighty840) [reply docs/docs/SDK Modules/KYC Onboarding.md:27]: There is already a link below to their handbook, which contains all the docs.

File: docs/docs/SDK Modules/KYC Onboarding.md (hunk 5 above)
Comment (mighty840) [reply docs/docs/SDK Modules/KYC Onboarding.md]: Added live and test instead of production and test

File: docs/docs/SDK Modules/KYC Onboarding.md (hunk 4 above)
Comment (mighty840) [reply docs/docs/SDK Modules/KYC Onboarding.md]: Modified the statement.
